app/models/book.rb

```diff
@@ -0,0 +1,2 @@
+class Book < ApplicationRecord
```

Comment:
Please validate the presence of the fields.

Reply:
Done